group-coordinator/src/main/java/org/apache/kafka/coordinator/group/classic/ClassicGroupState.java

```diff
@@ -114,15 +115,20 @@ public enum ClassicGroupState {
         DEAD.addValidPreviousStates(STABLE, PREPARING_REBALANCE, COMPLETING_REBALANCE, EMPTY, DEAD);
     }
 
-    ClassicGroupState(String name) {
+    ClassicGroupState(String name, String lowerCaseName) {
         this.name = name;
+        this.lowerCaseName = lowerCaseName;
     }
 
     @Override
     public String toString() {
         return name;
     }
 
+    public String toLowerCaseString() {
```

Comment:
nit: I wonder if we should also add `toLowerCaseString` to `ConsumerGroup.ConsumerGroupState` in order to be consistent.

Reply:
Actually, it would be great to do the same there. We could also capitatlize the states to be consistent too.

Reply:
Thanks for the suggestion, I added the  `toCapitalizedString` to `ConsumerGroup.ConsumerGroupState`.